client/src/main/java/se/fortnox/reactivewizard/client/RxClientProvider.java

```diff
@@ -54,11 +75,24 @@ public RxClientProvider(HttpClientConfig config, HealthRecorder healthRecorder)
             .pipelineConfigurator(UnsubscribeAwareHttpClientToConnectionBridge::configurePipeline);
 
         if (config.isHttps()) {
-            client = client.unsafeSecure();
+            return configureSsl(client, config.getHost(), config.getPort(), config.isUnsafeSecure());
         }
         return client;
     }
 
+    /**
+     * Allows for customization of the SSLEngine
+     *
+     * @return An configured SSLEngine
+     */
+    protected SSLEngine configureSslEngine(String host, int port) {
```

Comment:
private or package protected

Reply:
Private would not allow for customization so package protected is then preferred. Users would then need to put their variant of RxClientProvider under "our" namespace but I cannot motivate why that would be a bad thing - package protected it is!